### PR TITLE
IGAPP-592: Html support TüNews

### PR DESCRIPTION
**Pull request by f1sh1918**
Replaced title & content with page component. Added sanitizer to prevent xss attacks.

https://issues.tuerantuer.org/browse/IGAPP-592

This pull request belongs to an issue on our bugtracker.
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword IGAPP.

### CODE REVIEW

**Review by klinzo — APPROVED** [review]
Good work :muscle:

**Comment by klinzo** [comment]
Builds are currently failing because of prettier. You could fix this by running `yarn prettier --write .`.

File: web/src/modules/common/constants/news.js [hunk 1]
```diff
@@ -0,0 +1 @@
+export const lastUpdateDateFormat = 'LLL'
```
Comment (SteffiStuffel) [inline web/src/modules/common/constants/news.js:1]: think it would be good to add this file to flow checks.

**Review by SteffiStuffel — APPROVED** [review]
nice. :)

**Review by Taggotty — APPROVED** [review]
Looks fine to me 👍🏼

File: web/src/modules/common/components/Page.js [hunk 2]
```diff
@@ -53,7 +54,10 @@ const Page = ({
       {defaultThumbnailSrc && <Thumbnail alt='' src={defaultThumbnailSrc} srcSet={thumbnailSrcSet} />}
       <Caption title={title} />
       {children}
-      <RemoteContent dangerouslySetInnerHTML={{ __html: content }} onInternalLinkClick={onInternalLinkClick} />
+      <RemoteContent
+        dangerouslySetInnerHTML={{ __html: sanitize(content) }}
```
Comment (maxammann) [inline web/src/modules/common/components/Page.js:58]: I can no longer see the previous discusison on this. Do we want to remove the sanitization in the api-client? I think we should avoid it here and do all sanitisation in the api-client. That way we easily support native and web. Also there are other fields which could require sanitisation.

I'm open for both, just sanitisating twice it a little bit overkill.

**Review by maxammann — APPROVED** [review]

File: web/src/modules/common/components/Page.js (hunk 2 above)
Comment (klinzo) [reply web/src/modules/common/components/Page.js:58]: Removed it here and created a separate issue to check our currently sanitizing and add it/improve it wherever necessary,The cms team also mentioned that our current sanitizing is not good enough.
Issue: https://issues.tuerantuer.org/browse/IGAPP-619

**Pull request merged**